packages/front-end/styles/global.scss

```diff
@@ -3173,6 +3173,10 @@ td.disabled {
   cursor: not-allowed;
   background-color: var(--disabled-background) !important;
 }
+.form-control:focus {
+  border-color: #{colors.$gray-400};
+  box-shadow: 0px 0px 0px 1px var(--violet-8);
+}
```

Comment:
Need to remove this as I'm handling this in a separate PR [here](https://github.com/growthbook/growthbook/pull/3509).